config/services.yaml

```diff
@@ -65,6 +65,8 @@ services:
 
     App\Stats\Persister\PersisterInterface: '@App\Stats\Persister\RedisPersister'
 
+    App\DateProvider\DateTimeProviderInterface: '@App\DateProvider\CurrentDateTimeProvider'
```

Comment:
Maybe that's not necessary (just one implementation for this interface; Symfony should be able to handle it by itself).
I've added it just to prevent future addition to this interface.
Moreover I have a question: have we a ci step (or a test, or whatever) that builds the container? Because, if not, it would be useful to validate these configurations.

Reply:
Yep! In the circleCI we have a install of the dependecies and build the container to run tests and phpstan! ;)

Reply:
Ok, so I guess I can leave this here because that's still valid

> I've added it just to prevent future addition to this interface.